### PR TITLE
common setup for orchestration tests

### PR DESCRIPTION
**Pull request by turadg**
refs: #9087

## Description

While getting started on #9087 I thought it would be helpful to have less boilerplate in tests.

### CODE REVIEW

File: packages/orchestration/test/supports.ts [hunk 1]
```diff
@@ -1 +1,62 @@
...
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { withAmountUtils } from '@agoric/zoe/tools/test-utils.js';
+import { makeHeapZone } from '@agoric/zone';
+import { E } from '@endo/far';
+
 export { makeFakeLocalchainBridge } from '@agoric/vats/tools/fake-bridge.js';
+
+export const orcSetup = async t => {
+  t.log('bootstrap vat dependencies');
+  const rootZone = makeHeapZone();
```
Comment (dckc) [inline packages/orchestration/test/supports.ts:21]: Want to switch the root zone to a durable zone while we're at it?

**Review by dckc — APPROVED** [review]
yay

File: packages/orchestration/test/supports.ts (hunk 1 above)
Comment (turadg) [reply packages/orchestration/test/supports.ts:21]: Good question. I tried but many of the fakes these tests rely on do not support durability. I've documented in a comment.

Comment (dckc) [reply packages/orchestration/test/supports.ts:21]: I wondered about that. Thanks for giving it a try.

**Review by 0xpatrickdev — APPROVED** [review]

**Pull request merged**